ribbon-finance/experiments-site
# Front page

## Test
1. Run `yarn`
2. Run `yarn start`
3. Visually confirm that everything looks good 
4. Click "Connect MetaMask" button and connect account
